### PR TITLE
Anpo fix execution engine reals

### PR DESCRIPTION
**Pull request by anporumb**

### CODE REVIEW

**Review by mattdurak — APPROVED** [review]
:shipit:

File: win32/tests/reals_win32_ut/reals_win32_ut.c [hunk 1]
```diff
@@ -62,6 +64,7 @@ TEST_FUNCTION(check_all_c_pal_reals)
     REGISTER_SYNC_GLOBAL_MOCK_HOOK();
     REGISTER_ARITHMETIC_GLOBAL_MOCK_HOOK();
     REGISTER_UUID_GLOBAL_MOCK_HOOK();
+    REGISTER_EXECUTION_ENGINE_GLOBAL_MOCK_HOOK();
```
Comment (mattdurak) [inline win32/tests/reals_win32_ut/reals_win32_ut.c:67]: >REGISTER_EXECUTION_ENGINE_GLOBAL_MOCK_HOOK [](http://example.com/codeflow?start=4&length=42)

We need another test that validates that all of the reals are getting tested! /s

**Review by dcristoloveanu — APPROVED** [review]
:shipit:

File: win32/tests/reals_win32_ut/reals_win32_ut.c (hunk 1 above)
Comment (anporumb) [reply win32/tests/reals_win32_ut/reals_win32_ut.c:67]: ;)

**Pull request merged**